FirebaseTestProject/Models/User.swift

```diff
@@ -8,13 +8,15 @@
 import Foundation
 
 struct User: Equatable, FirebaseConvertable {
-    let id: UUID
+    let id: String
+    var imageURL: String
     var name: String
     
-    init(id: UUID = .init(), name: String) {
+    init(id: String, name: String, imageURL: String = "https://firebasestorage.googleapis.com/v0/b/instacademy-test1.appspot.com/o/images%2Fusers%2Fdefault_profile.png?alt=media&token=6700ffd7-2675-45ad-a197-bb2e4e307da7") {
```

Comment:
Good idea! I would implement this right away, but I'm unsure of how to reference an image from the app bundle from within the `User` model, since the model stores a URL. I know URLs can reference local files, but if a local file URL from one device is stored in Firebase, would it work on other devices as well?

Reply:
I believe the `Bundle` has a [`url(forResource:withExtension:)`](https://developer.apple.com/documentation/foundation/bundle/1411540-url) method that gives you the specific URL on a user's device for `defaultPhoto.png` or whatever.   I think that would probably work?

Reply:
It should work! I was skeptical about making it part of the `User` model since we don’t want a local file URL to be saved to Firebase by accident, but I doubt that’ll happen. I’ll push that change in a few minutes!